tests/dialects/onnx/test_broadcast.py

```diff
@@ -5,30 +5,46 @@
     unidirectional_broadcast_shape,
 )
 
-m_test_cases: list[tuple[tuple[list[int], list[int]], list[int]]] = [
-    (([2, 3, 4, 5], []), [2, 3, 4, 5]),
+m_test_cases: list[
+    tuple[
+        tuple[list[int | None] | None, list[int | None] | None], list[int | None] | None
+    ]
+] = [
+    (([2, 3, 4, 5], [None]), [2, 3, 4, 5]),
```

Comment:
I'm a little confused, why the `[None]`?

Reply:
I meant to add the failure cases with `None` as the result below the successful cases